LL/rig_alban.trig

```diff
@@ -271,8 +273,7 @@ _:missing-02dcec05
 <#Conaire-a4027f50>
     a foaf:Person;
     irishRel:nomName "Conaire";
-    rel:childOf <#Etersceoil>;
-    owl:sameAs <http://example.com/LL/corco_dubni.trig#Conaire>.
+    rel:childOf <#Etersceoil>.
```

Comment:
I think this is Conaire Mór. Compare http://example.com/Rawl_B502/genelach_ríg_n_alban.trig#ConaireMóir

Reply:
Yes, it is already linked as `owl:sameAs` from Rawl_B502/genelach_ríg_n_alban.trig.  `owl:sameAs` is symmetric so this will work but for avoidence of doubt, I have put it in.